### DISCUSSION TITLE
Starting with gradle

### DISCUSSION THREAD

**Discussion by SwordOfSouls** [Minestom/Minestom#1254] [Help] [answered]
I'm currently attempting to make my first MineStorm project with gradle and running into some issues. 
`plugins {
    id 'java'
    id "com.github.johnrengelman.shadow" version "6.1.0"
}

group 'org.swordofsouls'
version '1.0-SNAPSHOT'

repositories {
    mavenCentral()
    maven { url 'https://jitpack.io' }
}

dependencies {
    implementation 'com.github.Minestom:Minestom:d596992c0e'
}

jar {
    manifest {
        attributes 'Main-Class': 'org.swordofsouls.MineStorm.Server',
                "Multi-Release": true
    }
}`

Error:
`Server:main: Could not resolve com.github.Minestom:Minestom:d596992c0e.
Required by:
    project :

Possible solution:
 - Declare repository providing the artifact, see the documentation at https://docs.gradle.org/current/userguide/declaring_repositories.html
`
If anyone has any ideas on how to fix this please let me know, thanks

**Comment by SwordOfSouls** [comment]
Sorry githubs code formatting didn't seem to work

**Reply by muelleel** [reply]
Make sure code blocks which are going trough multiple lines have at the start and end three "`"
so "```"

```
example
multiple lines
    indentions
```

**Comment by jglrxavpok** [comment, marked as answer]
Make sure the Java version used for Gradle is at least Java 17.

**Reply by SwordOfSouls** [reply]
That seems to fix it, I thought gradle only supported Java 16 and below but it seems I was incorrect. Thanks for the help
